mollie/api/resources/subscriptions.py

```diff
@@ -0,0 +1,19 @@
+from ..objects.subscription import Subscription
+from .base import Base
+
+
+class Subscriptions(Base):
+    def get_resource_object(self, result):
+        return Subscription(result, self.client)
+
+    def create(self, data=None, **params):
+        raise NotImplementedError('Only the "list" endpoint is supported for Subscriptions.')
```

Comment:
More a style issue than a bug: this exception is prone to bitrot. When you're expanding this endpoint (f.i. adding support for 'delete', you'll need to adapt the error messages in all endpoints, which you're bound to forget.

It's also questionable if it's useful to inform an implementor that 'list' is available when all he wants to use is 'create' (Customer: "Hello, I would like a banana." Salesman: "we have vacuum cleaners." Customer: "But I want bananas!")

For improved style: just mention in the exception what isn't supported, not what other options you might have, except when they would help an implementor achieving the same result (f.i. `raise NotImplemented("Endpoint 'create' is not available, try 'CustomerSubscriptions.create()' instead")`).